Code/Tools/AssetProcessor/native/InternalBuilders/SettingsRegistryBuilder.cpp

```diff
@@ -307,12 +307,12 @@ namespace AssetProcessor
                     }
                     file.Close();
 
-                    // Hash all specializations tags
+                    // Hash only the launcher type and build config specializations tags
                     size_t hashedSpecialization{};
-                    for (size_t specIndex{}; specIndex < specialization.GetCount(); ++specIndex)
-                    {
-                        AZStd::hash_combine(hashedSpecialization, specialization.GetSpecialization(specIndex));
-                    }
+                    // Get the launcher type specialization tag
```

Comment:
I'm assuming 0 and 1 are the indices to look up the launcher type and build config? Is there a variable tracking that special index? Maybe just to replace the "magic" numbers.

Reply:
Yes, 0 is the launcher type and 1 is the build config.
I'll add constants for both.